docs/erp_fhirversion_change_20241101.adoc

```diff
@@ -0,0 +1,74 @@
+= E-Rezept FHIR Veränderungen für 01.11.2024 image:gematik_logo.png[width=150, float="right"]
+// asciidoc settings for DE (German)
+// ==================================
+:imagesdir: ../images
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+:toc: macro
+:toclevels: 3
+:toc-title: Inhaltsverzeichnis
+:AVS: https://img.shields.io/badge/AVS-E30615
+:PVS: https://img.shields.io/badge/PVS-C30059
+:FdV: https://img.shields.io/badge/FdV-green
+:eRp: https://img.shields.io/badge/eRp-blue
+:KTR: https://img.shields.io/badge/KTR-AE8E1C
+
+Ab dem 01.11.2024 ist das gematik Workflow Package 1.3 gültig. Es gibt sowohl einen neuen Endpunkt $dispense für die Apotheke als auch ein paar kleine Veränderungen an den FHIR-Objekten. Diese Seite bietet eine Zusammenfassung der Veränderungen in einem Format, das zeigt, was die verschiedenen Hersteller tun müssen, um ein bestimmtes Feature FHIR-mäßig umzusetzen. Alle Breaking Changes sind ebenfalls aufgelistet.
+
+toc::[]
+
+== Versionsübergabegänge
+Ab dem 01.11.2024 kann der E-Rezept-Fachdienst mehrere aktive Profile haben. Das bedeutet, dass Übergangszeiten mit mehr als zwei aktiven Profilen möglich sind. Vom 15.01.2025 bis 15.07.2025 sind die drei Profile 1.2, 1.3 und 1.4 gleichzeitig aktiv. In dieser Zeit wird besonders auf das meta.profile der Ressourcen geachtet, um festzustellen, gegen welche Profile validiert werden sollte.
+
+Unten sind sowohl ein Diagramm als auch eine Tabelle zu sehen, die zeigen, wann welches Profil aktiv ist und bis wann.
+
+image:puml_workflow_profile_version_timeline.png[]
+
+|===
+h|Version  h|Gültig ab  h|Gültig bis
+|Profile 1.2| - | 15.07.2025
+|Profile 1.3| 01.11.2024 | 15.07.2025
+|Profile 1.4| 15.01.2025 | tbd
+|Profile 1.5 (Vorläufig)| 15.07.2025 | -
+|===
+
+NOTE: Aufgrund der Updatezyklen von Apps im Feld können die FdVs die Profilversionen 1.2 und 1.3 im Zeitraum vom 15.04.2025 bis zum 15.07.2025 weiterhin nutzen. Ab dem 15.07.2025 ist auch für die FdVs die Verwendung der Versionen 1.2 und 1.3 nicht mehr möglich.
+
+NOTE: Ab dem 15.07.2025 ist Profil 1.4 nur für die FdVs verfügbar.
+
+== Änderungen
+Die Änderungen und Neuigkeiten sind hier aufgelistet, zuerst als Übersichtstabelle und danach im Detail. Sie sind auf Feature-Ebene aufgeteilt und es wird angegeben, welche Hersteller von den spezifischen Änderungen betroffen sind. Jedes Feature ist dann auch für die betroffenen Hersteller aufgeteilt. Das bedeutet, dass ein Hersteller den genauen Abschnitt finden kann, der beschreibt, was für ein bestimmtes Feature zu tun ist.
```

Comment:
Das würd eich prägnanter Formulieren. Z.b. "Im Folgenden finden sich die Änderungen des gematik Workflow Package 1.3 in Form einer Übersichtstabelle und detaillierten Beschreibungen, aufgeteilt nach Features und betroffenen Clientsystemen"